app/services/patient_import/spreadsheet_transformer.rb

```diff
@@ -221,7 +221,7 @@ def import_user
     end
 
     def patient_status(row)
-      row[:died].presence&.downcase.in?(["yes", "y"]) ? Patient.statuses["dead"] : Patient.statuses["active"]
+      row[:died]&.downcase&.in?(["yes", "y"]) ? Patient.statuses["dead"] : Patient.statuses["active"]
```

Comment:
Is this change meant for this PR?

Reply:
I guess it works. `"".downcase` should interpreted as "active".

Reply:
Yeah, it was, I was cleaning up a standardrb issue with this change.